src/main/include/RobotContainer.h

```diff
@@ -54,8 +84,25 @@ class RobotContainer {
     std::shared_ptr<Drivetrain> drivetrain_;
     std::shared_ptr<Intake> intake_;
 
-    // Commands
-    ExampleCommand m_autonomousCommand;
+    /**
+     * User interfaces
+     * - Gamepads
+     * - Joystick Buttons
+     */
+    std::shared_ptr<frc::Joystick> gamepad1_;
+    std::shared_ptr<frc::Joystick> gamepad2_;
 
-    void ConfigureButtonBindings();
+    std::shared_ptr<frc2::JoystickButton> driver_a_button_;
```

Comment:
I'll go ahead and make an issue to get all the other buttons put in the code. Could probably have that as a task for one of the new programmers

Reply:
Yeah, good first issue for sure